SoftLayer/CLI/block/modify.py

```diff
@@ -0,0 +1,70 @@
+"""Modify an existing block storage volume."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import exceptions
+
+
+CONTEXT_SETTINGS = {'token_normalize_func': lambda x: x.upper()}
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.argument('volume-id')
+@click.option('--new-size', '-c',
+              type=int,
+              help='New Size of block volume in GB. '
+                   '***If no size is specified, the original size of '
+                   'volume will be used.***\n'
+                   'Potential Sizes: [20, 40, 80, 100, 250, '
+                   '500, 1000, 2000, 4000, 8000, 12000] '
+                   'Minimum: [the size of the origin volume] '
+                   'Maximum: [the minimum of 12000 GB or '
+                   '10*(origin volume size)]')
```

Comment:
This 10x limit will only apply to volumes created before volume modification is released, so this comment may need to be adjusted to mention that. On the other hand, we don't want the help output to grow too large, so I wonder if it would be better to leave out the max here and rely on an error message if the size is too large.